crates/cast/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "cast"
+name = "foundry-cast"
```

Comment:
this conflicts with the crates.io package `cast`, used by criterion, which makes newer versions of rust-analyzer mad